### PR TITLE
{tools}[GCCcore/10.2.0,GCCcore/10.3.0] nvtop v1.2.2

### PR DESCRIPTION
**Pull request by crubb**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by migueldiascosta** [comment]
@boegelbot please test @ generoso

**Comment by migueldiascosta** [comment]
@boegelbot please test @ jsc-zen2

File: easybuild/easyconfigs/n/nvtop/nvtop-1.2.2-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+source_urls = ['https://github.com/Syllo/nvtop/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['d6ca7f467cc1048ebd1ebf7add753650c5d7c8868752c0c57ed2c85df62f5810']
+
+builddependencies = [
+    ('binutils', '2.35'),
+    ('CMake', '3.18.4'),
+]
+
+# Note: software dlopen's libraries from the OS installed drivers, thus no specific CUDA dependency.
```
Comment (migueldiascosta) [inline easybuild/easyconfigs/n/nvtop/nvtop-1.2.2-GCCcore-10.2.0.eb:20]: @crubb @Micket  would it make sense then to remove the CUDA dependency from the existing `nvtop-1.2.1-GCCcore-10.2.0.eb` easyconfig, for consistency with `nvtop-1.2.1-GCCcore-10.3.0.eb` and these new ones?

Comment (crubb) [reply easybuild/easyconfigs/n/nvtop/nvtop-1.2.2-GCCcore-10.2.0.eb:20]: Happy to oblige (see new commit), unless there are objections from @Micket? Glad to oblige (see new commit), unless there are objections from @Micket?

Comment (Micket) [reply easybuild/easyconfigs/n/nvtop/nvtop-1.2.2-GCCcore-10.2.0.eb:20]: no objections

**Comment by migueldiascosta** [comment]
Test report by @migueldiascosta
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
c2dhpc - Linux CentOS Linux 7.6.1810, x86_64, AMD EPYC 7601 32-Core Processor (zen), Python 2.7.5
See https://gist.github.com/b955a7fc4288e02d5af7d6b82f4f1fba for a full test report.

**Review by migueldiascosta — APPROVED** [review]
lgtm

**Comment by migueldiascosta** [comment]
Going in, thanks @crubb!

**Pull request merged**